src/background.ts

```diff
@@ -57,3 +61,17 @@ const firebase = initializeApp(JSON.parse(EnvVars.TEMPLE_FIREBASE_CONFIG));
 getMessaging(firebase);
 
 updateRulesStorage();
+
+async function prepareAppIdentity() {
+  const { privateKey, publicKey, publicKeyHash } = await generateKeyPair();
+
+  const ts = new Date().toISOString();
+
+  await putStoredAppInstallIdentity({
+    version: PackageJSON.version,
+    privateKey,
+    publicKey,
+    publicKeyHash: publicKeyHash.slice(0, 16),
```

Comment:
It would be better to store some amount of first and last characters because last ones are a checksum